__tests__/app.test.js

```diff
@@ -206,3 +206,14 @@ describe("DELETE /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: Deletes the specified comment and responds with no content", () => {
+    return request(app)
+      .delete("/api/comments/3")
+      .expect(204)
+      .then(({ body }) => {
+        expect(body).toEqual({});
+      });
+  });
+});
```

Comment:
What about tests for errors here? If I delete a comment that doesn't exist or an invalid id??